lib/sessions.ts

```diff
@@ -95,6 +95,11 @@ type SessionClaim = {
   expires_at: string;
   attributes: Attributes;
   authentication_factors: AuthenticationFactor[];
+  // Since custom claims are free-from JSON, the best we can say is that we collect them into an
+  // object with string keys.
+  //
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  custom_claims: Record<string, any>;
```

Comment:
Oops! In addition to the typo ("free-from" -> "free-form"), this field doesn't actually exist! Removing in #158